assessment_store/tasks/db_tasks.py

```diff
@@ -183,11 +183,40 @@ def seed_assessment_store_db(c, environment="local"):
             else:
                 _echo_print("No new scoring systems were added.")
 
-            # Associate scoring systems with all rounds
+            # Associate scoring systems with rounds
             round_ids = db.session.query(Round).with_entities(Round.id).all()
+            default_scoring_system_id = scoring_system_data[0]["id"]
+
+            specific_round_scoring = [
+                {
+                    "round_id": "ae223686-cbcc-4548-8b52-05898c315a59",
```

Comment:
do we want fund specific logic in the code here? Something about doing this here doesn't feel right but I'll have to have a look around to see where its otherwise configured. 

Looks like everything previously defaulted to 1 to 5 and now we want to be able to override

I'd be inclined to try and put this information here or something similar https://github.com/communitiesuk/funding-service-pre-award/blob/main/assessment_store/config/mappings/assessment_mapping_fund_round.py#L231-L240

Having it spread out between configuration and tasks code feels like an (even more slippery) slope

Reply:
https://github.com/communitiesuk/funding-service-pre-award/pull/148#pullrequestreview-2547565693